config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  
+  root 'users#index'
+  resources :users do
+    resources :posts
+  end
```

Comment:
- [ ] [OPTIONAL] Since we only have the `index` and the `show` actions for now. I will recommend that you limit the actions that can be performed on both users and posts by doing this:

``` ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```

It defines a nested resource relationship between users and posts, where only the index and show actions can be performed on both resources. This means that the routes generated by this code would be:

``` ruby
GET /users (index action)
GET /users/:id (show action)
GET /users/:user_id/posts (index action)
GET /users/:user_id/posts/:id (show action)
```

### On the other hand, your route:
defines a root route for the application to the index action of the UsersController. Additionally, it also defines a nested resource relationship between users and posts, where all seven RESTful actions can be performed on both resources. This means that the routes generated by this code would be:

``` ruby
GET / (root route, maps to the index action of the UsersController)
GET /users (index action)
POST /users (create action)
GET /users/new (new action)
GET /users/:id (show action)
GET /users/:id/edit (edit action)
PATCH/PUT /users/:id (update action)
DELETE /users/:id (destroy action)
GET /users/:user_id/posts (index action)
POST /users/:user_id/posts (create action)
GET /users/:user_id/posts/new (new action)
GET /users/:user_id/posts/:id (show action)
GET /users/:user_id/posts/:id/edit (edit action)
PATCH/PUT /users/:user_id/posts/:id (update action)
DELETE /users/:user_id/posts/:id (destroy action)
```

- Let's adopt my recommendation since we only have two actions in this milestone 💪🏽